src/soulmates/index.js

```diff
@@ -0,0 +1,29 @@
+import config from '../_shared/js/config';
+import { getIframeId, getWebfonts, resizeIframeHeight } from '../_shared/js/messages';
+import { write } from '../_shared/js/dom';
+import { portify } from '../_shared/js/dev';
+
+const cardContainer = document.getElementsByClassName("adverts__row")[0];
+
+getIframeId()
+  .then(({host}) => fetch(`${portify(host)}${config.soulmatesUrl}`))
+  .then(response => response.json())
+  .then(soulmates => soulmates.map(createSoulmateCard))
+  .then(cards => addSoulmatesCards(cards))
```

Comment:
You can merge this line with the next one using `Promise.all`, this will go a little bit faster.

```
.then(cards => Promise.all([addSoulmatesCards(cards), getWebfonts()])
```